### PR TITLE
Fix missing copyright change for SANDIA CORP -> NTESS

### PR DESCRIPTION
**Pull request by crtrott**
"Cherry-pick" of #2799 into `master`

### CODE REVIEW

**Comment by ndellingwood** [comment]
Should tag in the CHANGELOG be updated as well?

**Comment by ndellingwood** [comment]
The CHANGELOG link should work once the tag is added

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Change Log
 
-## [2.9.99](https://github.com/kokkos/kokkos/tree/2.9.99) (2020-01-27)
-[Full Changelog](https://github.com/kokkos/kokkos/compare/2.9.00...2.9.99)
+## [3.0.00](https://github.com/kokkos/kokkos/tree/3.0.0) (2020-01-27)
```
Comment (dalg24) [inline CHANGELOG.md]:
```suggestion
## [3.0.00](https://github.com/kokkos/kokkos/tree/3.0.00) (2020-01-27)
```
@ndellingwood

Comment (ndellingwood) [reply CHANGELOG.md]: Good catch

**Comment by crtrott** [comment]
Fixed

**Pull request merged**